### PR TITLE
Refactor TextEditor: Move bubble menu toolbar in separate component

### PR DESCRIPTION
**Pull request by annano**
Screenshot from 2023-08-31 15-02-19
Screenshot from 2023-08-31 15-02-28

# Contribution checklist

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [ ] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [ ] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ cd ./dev/
 rush build    # Will build all the required packages.
 rush bundle   # Will prepare bundles.
 rush docker:build   # Will build Docker containers for all applications in the local Docker environment.
-docker-compose up -d --force-recreate # Will set up all the containers
+docker compose up -d --force-recreate # Will set up all the containers
```
Comment (mixerka) [inline README.md]: 👀

Comment (haiodo) [reply README.md]: Better to switch to new docker compose, https://docs.docker.com/compose/reference/ in all places, not just here and do it in separate PR.

**Comment by haiodo** [comment]
I'd like idea to extract toolbar from editor, @mixerka could you please review, thanks.

File: README.md (hunk 1 above)
Comment (annano) [reply README.md]: Reverted

**Review by mixerka — APPROVED** [review]
I think, it's an awesome refactoring

**Review by haiodo — APPROVED** [review]

**Pull request merged**